index.html

```diff
@@ -6,10 +6,88 @@
     <title>Tools and Workflow for the Web - Introduction to GitHub Projects</title>
     <link rel="stylesheet" href="css/styles.css" type="text/css" />
   </head>
-  <body>
-    <h1>Hello, my name is Jesse.</h1>
-    <p>My favourite color is grey.</p>
-    <p>My favourite food is pho.</p>
-    <p>My favourite hobby is programming.</p>
-  </body>
```

Comment:
The elimination of this code snippet was expedient as there were no sectioning tags.